pyramid_oereb/contrib/data_sources/standard/hook_methods.py

```diff
@@ -1,182 +1,75 @@
 # -*- coding: utf-8 -*-
-import datetime
 import logging
 import binascii
 
-from mako import exceptions
-from mako.template import Template
-from pyramid.httpexceptions import HTTPNotFound
-from pyramid.path import AssetResolver
-from pyramid.response import Response
+from pyramid.httpexceptions import HTTPNotFound, HTTPServerError
 from sqlalchemy import cast, Text
 
 from pyramid_oereb.contrib.data_sources.standard.sources.plr import StandardThemeConfigParser
-from pyramid_oereb import Config, database_adapter, route_prefix
+from pyramid_oereb import database_adapter
 from pyramid_oereb.core import b64
 from pyramid_oereb.core.records.image import ImageRecord
-from pyramid_oereb.core.records.office import OfficeRecord
 
 
 log = logging.getLogger(__name__)
 
 
-def get_symbol(request):
```

Comment:
Regarding https://github.com/openoereb/pyramid_oereb/issues/1476: Is the idea to reorganize the hook_methods in this PR and  to address the symbol issue in a next one?

Reply:
Yes-we will do this later to not get a too big PR.

Reply:
@michmuel why are you requesting changes?